devops/resources/user-retirement-archiver.sh

```diff
@@ -19,9 +19,9 @@ assume-role ${ROLE_ARN}
 cd $WORKSPACE/tubular
 pip install -r requirements.txt
 
-# In case this is being run without an explicit END_DATE, default to running with "now"
-if [[ "$END_DATE" == "" ]]; then
-    END_DATE=$(date +%Y-%m-%d)
+# In case this is being run without an explicit END_DATE, default to running with "now" - COOL_OFF_DAYS
+if [[ ! -v END_DATE ]]; then
+    END_DATE=$(date --date "$(date "+%B %d, %Y") -$COOL_OFF_DAYS days" +%y-%m-%d)
```

Comment:
naive question -- where does COOL_OFF_DAYS come from?   And will it always be defined?

Reply:
COOL_OFF_DAYS is a job parameter with a default value: https://github.com/edx/jenkins-job-dsl/blob/47b9a3d63670ee237faca055c56f05d909172752/platform/jobs/RetirementJobs.groovy#L251

Seems okay to assume it exists.